File: Python/fibonacci.py
def fibonacci_Seq(number):
    if number <= 1:
        return number
    else:
        return fibonacci_Seq(number - 1) + fibonacci_Seq(number - 2)


my_input = int(input("Enter Your Fibonacci Number: "))
my_fibonacci_list = []

for i in range(my_input):
    my_fibonacci_list.append(fibonacci_Seq(i))

if my_input <= 0:
    print("The Number is Wrong.")
else:
    print(f"Fibonacci Sequence: {my_fibonacci_list}")

File: Go/fibonacci.go
package main

import (
	"fmt"
)

func fibonacciSeq(number int) int {
	if number <= 1 {
		return number
	} else {
		return fibonacciSeq(number-1) + fibonacciSeq(number-2)
	}
}

func main() {
	var myInput int
	fmt.Print("Enter Your Fibonacci Number: ")
	fmt.Scan(&myInput)

	var myFibonacciList []int

	for i := 0; i < myInput; i++ {
		myFibonacciList = append(myFibonacciList, fibonacciSeq(i))
	}

	if myInput <= 0 {
		fmt.Println("The Number is Wrong.")
	} else {
		fmt.Printf("Fibonacci Sequence: %v\n", myFibonacciList)
	}
}